packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -515,8 +516,7 @@ type IdResolver = (
 
 function globSafePath(path: string) {
   // slash path to ensure \ is converted to / as \ could lead to a double escape scenario
-  // see https://github.com/mrmlnc/fast-glob#advanced-syntax
-  return fg.escapePath(normalizePath(path))
+  return escapePath(normalizePath(path))
```

Comment:
wouldn't `convertPathToPattern` do the same without needing `normalizePath`? except `convertPathToPattern` only converts backslashes if executed on windows

Reply:
`normalizePath` uses `slash` which also handles the extended path length edge case, so I think it's a tiny bit safer to keep that as is.